pkg/kritis/review/review_test.go

```diff
@@ -222,13 +222,15 @@ func TestReviewGAP(t *testing.T) {
 			hasRequiredAtts: false,
 			shouldErr:       false,
 		},
+		// TODO: After payload check implemented in cryptolib,
+		// `hasRequiredAtts` should be false, and `shouldErr` should be true.
 		{
 			name:            "image with invalid attestation",
 			image:           img,
 			policies:        oneGAP,
 			attestations:    invalidAtts,
-			hasRequiredAtts: false,
-			shouldErr:       true,
+			hasRequiredAtts: true,
```

Comment:
Can we update the PR so that it doesn't change existing test behavior?

Reply:
Reverted to original behavior.